samples/gpio-pci-idio-16.c

```diff
@@ -158,14 +159,17 @@ main(int argc, char *argv[])
         .write_data = &migration_write_data
     };
 
-    while ((opt = getopt(argc, argv, "Rv")) != -1) {
+    while ((opt = getopt(argc, argv, "Rvm")) != -1) {
         switch (opt) {
             case 'R':
                 restart = false;
                 break;
             case 'v':
                 verbose = true;
                 break;
+            case 'm':
```

Comment:
total nit, but can we make it "M" so it's like capital-R to disable? And alphabetical order?

Reply:
sure